repos/exporter/lib/handler.ts

```diff
@@ -135,6 +140,24 @@ export const handler: Handler<ExportLambdaBody> = async ({ paths, noc, exportPre
         }),
     );
 
+    const date = new Date();
+    const numberOfExpectedNetexFiles = paths.length;
+    const metadata = { date, numberOfExpectedNetexFiles };
+
+    console.log(
+        `putting metadata: date - ${metadata.date.toDateString()} numberOfExpectedNetexFiles - ${
+            metadata.numberOfExpectedNetexFiles
```

Comment:
Do we still want this here?

Reply:
@RobCattonKPMG  it can be good for logging to have it ?